komodor/manifest.json

```diff
@@ -0,0 +1,31 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "995fe904-e761-4f2f-8dbf-148baf3f080a",
+  "app_id": "komodor",
+  "display_on_public_website": false,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Track changes across your entire K8s landscape and stack",
+    "title": "Komodor Automation",
+    "classifier_tags": [
+      "Supported OS::Linux",
+      "Supported OS::Mac OS",
+      "Supported OS::Windows",
+      "Offering::Integration"
+    ]
+	 },
```

Comment:
I think we're almost there! The media field is missing here, and you seem to have an earlier version where you did have references to the image files included in the repo, you may add them back in like this here:

```suggestion
    ],
    "media": [
      {
        "media_type": "image",
        "caption": "Main services screen.",
        "image_url": "images/Komodor_screen_01.png"
      },
      {
        "media_type": "image",
        "caption": "Service view events timeline and related services.",
        "image_url": "images/Komodor_screen_02.png"
      },
      {
        "media_type": "image",
        "caption": "Services view reviewing a deployment and its changes.",
        "image_url": "images/Komodor_screen_03.png"
      },
      {
        "media_type": "image",
        "caption": "Reviewing a deployment's replicaset and its pods and logs",
        "image_url": "images/Komodor_screen_04.png"
      },
      {
        "media_type": "image",
        "caption": "Events timelines for multiple clusters and deployments.",
        "image_url": "images/Komodor_screen_05.png"
      },
      {
        "media_type": "image",
        "caption": "Service view showing Datadog Monitoring Alert.",
        "image_url": "images/Komodor_screen_06.png"
      },
      {
        "media_type": "image",
        "caption": "Datadog metrics view showing link back into Komodor.",
        "image_url": "images/Komodor_screen_07.png"
      }
    ]
  },
```

Reply:
applied, Thanks!